src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,40 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+
+import './TodoInfo.scss';
+
+import { UserInfo } from '../UserInfo';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+    id,
+  } = todo;
+
+  return (
+    <article
+      className={classNames(
+        'TodoInfo',
+        {
+          'TodoInfo--completed': completed,
+        },
+      )}
+      data-id={id}
+    >
+      <h2 className="TodoInfo__title">
```

Comment:
Consider using a more descriptive class name for better readability and maintainability. 🤓